archive/backend-backups/main_inmemory_backup_1753734039.py

```
@@ -0,0 +1,1692 @@
+    if current_user.role == UserRole.HR:
+        return list(database["properties"].values())
+    elif current_user.role == UserRole.MANAGER:
+        return [p for p in database["properties"].values() if current_user.id in p.manager_ids]
+    else:
+        raise HTTPException(status_code=403, detail="Access denied")
+
+@app.post("/properties/{property_id}/managers")
+async def assign_manager(
+    property_id: str,
+    manager_email: EmailStr,
+    current_user: User = Depends(get_current_user)
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)